baseapp/pom.xml

```diff
@@ -12,16 +12,37 @@
     <artifactId>baseapp</artifactId>
     <packaging>jar</packaging>
 
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <dropwizard.version>3.0.1</dropwizard.version>
+        <guice.version>5.1.0</guice.version>
```

Comment:
it's better to call it `dep.guice.version` and `dep.dropwizard.version` to keep it consistent with other properties in airlift and Trino

Reply:
Please check the latest commit. I am using the standard versions from airbase.